### DISCUSSION TITLE
New Computer

### DISCUSSION THREAD

**Discussion by TheMrThx** [amidaware/tacticalrmm#1619] [Q&A] [answered]
Hi, 
Is any way to set task for when you add new computer? 
Like when new agent is instaled then install software x

**Comment by dinger1986** [comment]
in your automation policy, create the task with a date in the past, then when a new agent is added, trmm will go to create that task, notice the date is in past, and create it with the current time + 5 minutes. so it will run basically 5 minutes after new a agent is added. for the task options make sure the schedule is set to "run once" and the "run asap after missed" is checked

**Reply by TheMrThx** [reply]
![image](https://github.com/amidaware/tacticalrmm/assets/54869030/64964dc5-7dfb-4a39-8ca2-7f268a05fa02)

I setup task like on ss but after sync with new computer task newer run

**Reply by silversword411** [reply]
Try setting the expires after date too. Far into the future I believe.

**Reply by TheMrThx** [reply]
Setting after not working but setting in past work but after long time like 10h. Is any other way?

**Reply by dinger1986** [reply, marked as answer]
write a check to check if certain software is installed if that fails then the task is to install whatever software.

So your check could check if certain software is installed, if it isnt the check will fail then that triggers the task to install whatever software
